test/any_sender_of_test.cpp

```diff
@@ -20,6 +20,7 @@
 #include <unifex/ready_done_sender.hpp>
 #include <unifex/transform.hpp>
 #include <unifex/inline_scheduler.hpp>
+#include <unifex/any_scheduler.hpp>
```

Comment:
Is this `#include` needed here?

Reply:
Yes.

Reply:
Where is it needed? There was no other change to this file to add a usage of `any_scheduler` that I could see...
Is there some transitive dependency of the code under test below that requires `any_scheduler`?

Reply:
Ah, I see there seems to be some kind of cyclic dependency between `any_scheduler` and `any_sender_of`...